library/registry

```diff
@@ -1,12 +1,12 @@
-# maintainer: Joffrey F <joffrey@docker.com> (@shin-)
-# maintainer: Sam Alba <sam@dotcloud.com> (@samalba)
 # maintainer: Stephen Day <stephen.day@docker.com> (@stevvooe)
 # maintainer: Olivier Gambier <olivier.gambier@docker.com> (@dmp42)
+# maintainer: Richard Scothern <richard.scothern@gmail.com>
 
 latest: git://github.com/docker/docker-registry@0.9.1
 0.8.1: git://github.com/docker/docker-registry@0.8.1
 0.9.1: git://github.com/docker/docker-registry@0.9.1
 
-2: git://github.com/docker/distribution@62b70f951f30a711a8a81df1865d0afeeaaa0169
-2.0: git://github.com/docker/distribution@62b70f951f30a711a8a81df1865d0afeeaaa0169
+2: git://github.com/docker/distribution@1341222284b3a6b4e77fb64571ad423ed58b0d34
+2.0: git://github.com/docker/distribution@1341222284b3a6b4e77fb64571ad423ed58b0d34
 2.0.0: git://github.com/docker/distribution@62b70f951f30a711a8a81df1865d0afeeaaa0169
```

Comment:
@stevvooe  How could this file be an immutable description of the contents of the image if the `Dockerfile` contains something like `FROM golang:1.4`, which is a floating tag (which derives from `buildpack-deps:jessie-scm`, another floating tag)? The goal is not to have an immutable description of the image, but rather a repeatable build to whatever extent possible. Mandating an immutable git tag is one of the few things that can reduce the unpredictability of the image build.